src/backend/tools/google_drive/sync/activity.py

```diff
@@ -24,6 +23,7 @@
 )
 from backend.tools.google_drive.tool import GoogleDrive
 
+RESTORE_ACTION_NAME="restore"
```

Comment:
can move inside the match block for restore

Reply:
feel like it makes sense to declare the const here. not sure what u mean by moving into the block, like just passing the string directly in the param?

will leave for now to merge pr. lmk if u want me to follow up